app/tests/e2e/test_connections.py

```diff
@@ -195,56 +201,22 @@ async def test_get_connection_by_id(
 @pytest.mark.anyio
 async def test_delete_connection(
     bob_member_client: RichAsyncClient,
-):
-    invitation_response = await bob_member_client.post(f"{BASE_PATH}/create-invitation")
-    invitation = invitation_response.json()
-    connection_id = invitation["connection_id"]
-
-    response = await bob_member_client.delete(f"{BASE_PATH}/{connection_id}")
-    assert response.status_code == 204
-
-    with pytest.raises(HTTPException) as exc:
-        response = await bob_member_client.get(f"{BASE_PATH}/{connection_id}")
-    assert exc.value.status_code == 404
-
-
-@pytest.mark.anyio
-async def test_bob_and_alice_connect(
-    bob_member_client: RichAsyncClient,
     alice_member_client: RichAsyncClient,
 ):
-    invitation_response = await bob_member_client.post(
-        f"{BASE_PATH}/create-invitation",
-    )
-    invitation = invitation_response.json()
+    connection_alias = "TempAliceBobConnectionDelete"
 
-    accept_response = await alice_member_client.post(
-        f"{BASE_PATH}/accept-invitation",
-        json={"invitation": invitation["invitation"]},
-    )
-    connection_record = accept_response.json()
-
-    assert await check_webhook_state(
-        client=alice_member_client,
-        topic="connections",
-        state="completed",
-        filter_map={
-            "connection_id": connection_record["connection_id"],
-        },
+    bob_and_alice_connection = await create_bob_alice_connection(
+        alice_member_client, bob_member_client, alias=connection_alias
     )
 
-    alice_connection_id = connection_record["connection_id"]
-    bob_connection_id = invitation["connection_id"]
+    bob_connection_id = bob_and_alice_connection.bob_connection_id
 
-    bob_connection = (
-        await bob_member_client.get(f"{BASE_PATH}/{bob_connection_id}")
-    ).json()
-    alice_connection = (
-        await alice_member_client.get(f"{BASE_PATH}/{alice_connection_id}")
-    ).json()
+    response = await bob_member_client.delete(f"{BASE_PATH}/{bob_connection_id}")
+    assert response.status_code == 204
 
-    assert "completed" in alice_connection["state"]
-    assert "completed" in bob_connection["state"]
+    with pytest.raises(HTTPException) as exc:
+        response = await bob_member_client.get(f"{BASE_PATH}/{bob_connection_id}")
+    assert exc.value.status_code == 404
```

Comment:
We used to delete for bob and alice. Maybe just add some comments that record is now deleted for Alice too. Then, to validate that, have a 404 assert if alice tries to delete her connection after bob has deleted.